### PR TITLE
Link to redirect after clicking in the finish shopping button

### PR DESCRIPTION
**Pull request by lucasayb**
#### What is the purpose of this pull request?
Link to redirect after clicking in the finish shopping button

#### What problem is this solving?
Link to redirect after clicking in the finish shopping button

#### How should this be manually tested?
Link to redirect after clicking in the finishe shopping button

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by regis-samurai — APPROVED** [review]

File: messages/en.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "admin/editor.minicart.description": "A simple mini cart that shows all products added to it",
     "admin/editor.minicart.showDiscount.title": "Show the quantity of saved money",
     "admin/editor.minicart.labelMiniCartEmpty.title": "Text to appear when the mini cart is empty",
+    "admin/editor.minicart.linkButtonFinishShopping.title": "Link to redirect after clicking in the finishe shopping button",
```
Comment (klzns) [inline messages/en.json]:
```suggestion
    "admin/editor.minicart.linkButtonFinishShopping.title": "Link to redirect after clicking in the Go to Checkout button",
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
### Added
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/index.js [hunk 3]
```diff
@@ -535,8 +537,14 @@ EnhancedMinicart.schema = {
...
       isLayout: false,
     },
+    linkButtonFinishShopping: {
+      title: 'admin/editor.minicart.linkButtonFinishShopping.title',
+      type: 'string',
+      isLayout: false,
+    },
     labelButtonFinishShopping: {
       title: 'admin/editor.minicart.labelButtonFinishShopping.title',
+      default: '/checkout/#/cart',
```
Comment (jgfidelis) [inline react/index.js]: wrong, this should be above, in the `linkButtonFinishShopping`

**Review by jgfidelis — CHANGES_REQUESTED** [review]
just one comment

File: react/index.js (hunk 3 above)
Comment (lucasayb) [reply react/index.js]: Changed. Thanks!

File: messages/en.json (hunk 1 above)
Comment (lucasayb) [reply messages/en.json]: Fixed the text

**Review by klzns — APPROVED** [review]

**Review by jgfidelis — APPROVED** [review]

**Comment by klzns** [comment]
@allcontributors please add @lucasayb for code

**Pull request merged**